src/Media.tsx

```diff
@@ -216,6 +217,8 @@ export interface MediaContextProviderProps<M> {
    * will be triggered, which could lead to unintended side-effects.
    */
   disableDynamicMediaQueries?: boolean
+
+  isPending?: boolean
```

Comment:
So this `isPending` prop can override the one that's generated from `useTransition()`?

Reply:
Nope, they are the same, generated in ResponsiveProvider and passed to Media. It reflects the first dependence where renderChildren originates from (the state update of mediaQueryMatches), even when renderChildren is decided later in the Media component. See also https://github.com/artsy/fresnel/blob/ed51bcea1973a2e68e2c7e10747955bc71f5a44c/src/DynamicResponsive.tsx#L150